quiz.js

```diff
@@ -102,9 +103,9 @@ if (numCorrect === 5) {
     resultsContainer.innerHTML = `Pretty Good! You got ${numCorrect} out of ${myQuestions.length} correct!`;
 } else if (numCorrect === 3) {
     resultsContainer.innerHTML = `Not bad! You got ${numCorrect} out of ${myQuestions.length} correct!`;
-} else if (numCorrect => 2)
+} else if (numCorrect => 2) // Comment
     resultsContainer.innerHTML = `Yikes! You got ${numCorrect} out of ${myQuestions.length} correct.`;
-} 
+} // End comment section
```

Comment:
This bracket `{` looks like it's the closing bracket for the `else if` but it's actually the closing bracket for the `showResults()` function.